### PR TITLE
Fix cygwin page style and general dev tools import

### PR DESCRIPTION
**Pull request by MaEtUgR**
While reading through the instructions to check I found some severe style issues on the cygwin page which again come from gitbook interpreting markdown quite a bit different. This prs main goal is to fix this.

Also I found that:
- The dev tools addition is importent on every windows page, I switched to only the general page
- Linux page duplicates the dev tools information, I switched to the imported txt and move the note to the end
- Dev tools page does not mention VC Code and eclipse which are very commonly used IDEs for PX4, I added them

Preview: https://maetugr.gitbooks.io/test/content/en/setup/dev_env.html

### CODE REVIEW

File: en/setup/_addition_dev_tools.txt [hunk 1]
```diff
@@ -10,6 +10,8 @@ Download and install the [QGroundControl Daily Build](https://docs.qgroundcontro
 
 The development team often use:
 
+* [Visual Studio Code](https://code.visualstudio.com/): quite new, popular open source IDE
+* [eclipse for C/C++](https://www.eclipse.org/downloads/eclipse-packages/): very feature rich Java based IDE
```
Comment (hamishwillee) [inline en/setup/_addition_dev_tools.txt]: eclipse -> Eclipse

These are fine if these are genuinely used by the team

File: en/setup/dev_env_linux.md [hunk 2]
```diff
@@ -91,24 +91,7 @@ Note:
 * ROS is installed with Gazebo7 by default (we have chosen to use the default rather than Gazebo8 or Gazebo9 to simplify ROS development).
 * Your catkin (ROS build system) workspace is created at **~/catkin_ws/**.
 
+<!-- import docs for other tools and next steps. -->
+{% include "_addition_dev_tools.txt" %}
```
Comment (hamishwillee) [inline en/setup/dev_env_linux.md]: The last part of the `_addition_dev_tools.txt` is a section next steps, which sends you to build, then you have the Note below. I'm not sure that works. Could we move the Note below into the _addition_dev_tools.txt ?

Comment (hamishwillee) [reply en/setup/dev_env_linux.md]: We might need to say "on linux" in the note

**Comment by hamishwillee** [comment]
Looks good. Couple of minor points. Have a look and then I'll do a merge and final tidy as post process.

File: en/setup/_addition_dev_tools.txt (hunk 1 above)
Comment (MaEtUgR) [reply en/setup/_addition_dev_tools.txt]: I wanted to write it like in their logo:
eclipse

File: en/setup/dev_env_linux.md (hunk 2 above)
Comment (MaEtUgR) [reply en/setup/dev_env_linux.md]: Exactly, on linux is necessary, then it's a better solution.

File: en/setup/_addition_dev_tools.txt (hunk 1 above)
Comment (MaEtUgR) [reply en/setup/_addition_dev_tools.txt]: fixed

File: en/setup/dev_env_linux.md (hunk 2 above)
Comment (MaEtUgR) [reply en/setup/dev_env_linux.md]: done

Comment (MaEtUgR) [reply en/setup/dev_env_linux.md]: I still think the content of `_addition_dev_tools.txt` should not be duplicate in the output but rather apear on its own page and get linked to. Or is that against your concept?

Comment (hamishwillee) [reply en/setup/dev_env_linux.md]: Depends on how you expect people to read the docs. I take your point though - pulled out the devtools in #550

**Pull request merged**